library/photon

```diff
@@ -11,17 +11,17 @@ GitCommit: 6e5f8985ee8c59ab061e8098612f6f3335359327
 arm64v8-GitFetch: refs/heads/aarch64/4.0-20211112
 arm64v8-GitCommit: 1f98e65320888217e46b4776e2c646e3c21420b1
 
-Tags: 3.0, 3.0-20211105
+Tags: 3.0, 3.0-20211119
 Architectures: amd64, arm64v8
-GitFetch: refs/heads/x86_64/3.0-20211105
-GitCommit: 85635c4557b95a12ab72d947033399e4fb7a795f
-arm64v8-GitFetch: refs/heads/aarch64/3.0-20211105
-arm64v8-GitCommit: f2c65914596e8e235bf1b398a1c8e35514d8d03c
+GitFetch: refs/heads/x86_64/3.0-20211119
```

Comment:
It looks like most of these new branches are unavailable in the git repo. Perhaps they failed to push to the git repo because you've run into the github repo limit? Deleting older branches should get github to allow pushing new ones.

Reply:
CC @frapposelli @YustasSwamp

Reply:
It is a known issue about our gerrit-photon account need F2A verification. We just fixed. And we have a new PR today.

Reply:
@frapposelli, please help to grant gerrit-photon write permission.